Add SASMA.ipynb and update README.md

kenty-k opened the pull request:
Summary
===
Added a new Jupyter Notebook file SASMA.ipynb.
Updated the README with information relevant to the research paper.
Branch name is add-sasma.


Review Request
===
Please check if the content of SASMA.ipynb is accurate.
Please check if the information in the README is appropriate.

kenty-k committed:
add SASMA.ipynb and update README

Yuki-11 commented on README.md:
```diff
@@ -49,6 +49,10 @@ python inference.py \
 
+## Training preprocessing
+For the training preprocesing , our proposed method# Assume shadow within the range of 40~100 uses SASMA.
+the code for SASMA is SASMA.ipynb. 
```
The first letter must be capitalized. Then please add the following text.

> For more information on SASMA, see the Citation section.

Yuki-11 replied:
The citation information will be added by me after I approve this pull request.

Yuki-11 commented:
@kenty-k 
Thanks for your contributions! Please fix my request. 
Best,

kenty-k committed:
Correct typos in README and edit SASMA.ipynb

kenty-k commented:
@Yuki-11 
I have addressed your feedback and made the necessary changes.
- Edited lines 53 and 54 of README.md.
- Removed unnecessary comments and logs from SASMA.ipynb.
Please confirm that

kenty-k committed:
Recorrect README

Yuki-11 commented:
Thanks!
